wiki/Glossary/en.md

```diff
@@ -384,19 +368,22 @@ There are also the Silver S and Silver SS grades, which are, respectively, an S
 
 ### Grid snapping
 
-Grid snapping is a button on the right side of the editor screen. Enabling it will force every newly placed (or existing, but moved) object to snap along the intersections of the grid. The level of sparseness can be adjusted by pressing [G] or by going to View » Grid Level. Grid snapping can help with aligning objects to each other and arranging them in patterns.
+Grid snapping is a button on the right side of the editor screen.
```

Comment:
"Grid snapping refers to a same-named button on the right hand side of the editor screen."